packages/web-components/fast-components/src/select/select.pw.spec.ts

```diff
@@ -0,0 +1,253 @@
+import type {
+    ListboxOption as FASTOption,
+    Select as FASTSelectType,
+} from "@microsoft/fast-foundation";
+import { expect } from "chai";
+import type { ElementHandle } from "playwright";
+import type { FASTDesignSystemProvider } from "../design-system-provider";
+
+type FASTSelect = HTMLElement & FASTSelectType;
+
+describe("FASTSelect", function () {
+    beforeEach(async function () {
+        if (!this.page && !this.browser) {
+            this.skip();
+        }
+
+        this.documentHandle = await this.page.evaluateHandle(() => document);
+
+        this.providerHandle = (await this.page.$("#root")) as ElementHandle<
+            FASTDesignSystemProvider
+        >;
+
+        this.setupHandle = await this.page.evaluateHandle(
+            ([document, provider]) => {
+                const element = document.createElement("fast-select") as FASTSelect;
+
+                for (let i = 1; i <= 3; i++) {
+                    const option = document.createElement("fast-option") as FASTOption;
+                    option.value = `${i}`;
+                    option.textContent = `option ${i}`;
+                    element.appendChild(option);
+                }
+
+                provider.appendChild(element);
+            },
+            [this.documentHandle, this.providerHandle] as [
+                ElementHandle<Document>,
+                ElementHandle<FASTDesignSystemProvider>
+            ]
+        );
+    });
+
+    afterEach(async function () {
+        if (this.setupHandle) {
+            await this.setupHandle.dispose();
+        }
+    });
+
+    // FASTSelect should render on the page
```

Comment:
when commenting on `FASTSelect` I would probably add a space to be `FAST Select` for better readability.  That said, these comments may not be needed at all if you add a little more verbosity to the test statement.

Reply:
FASTSelect is the name of the symbol - it is referencing a specific thing